cmd/manager/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/docker/infrakit/cli"
+	"github.com/docker/infrakit/discovery"
+	"github.com/docker/infrakit/leader"
+	"github.com/docker/infrakit/manager"
+	"github.com/docker/infrakit/rpc"
+	group_rpc "github.com/docker/infrakit/rpc/group"
+	"github.com/docker/infrakit/store"
+	"github.com/spf13/cobra"
+)
+
+type backend struct {
+	id         string
+	plugins    discovery.Plugins
+	leader     leader.Detector
+	snapshot   store.Snapshot
+	pluginName string //This is the name of the stateless group plugin that the manager will proxy for.
+}
+
+func main() {
+
+	logLevel := cli.DefaultLogLevel
+	backend := &backend{}
+
+	cmd := &cobra.Command{
+		Use:   filepath.Base(os.Args[0]),
+		Short: "Manager",
+		PersistentPreRun: func(c *cobra.Command, args []string) {
+			cli.SetLogLevel(logLevel)
+		},
+		PersistentPostRunE: func(c *cobra.Command, args []string) error {
+			return runMain(backend)
```

Comment:
Incremental readability improvement on my end - eliminate the `PersistentPostRunE` and have the subcommands call `runMain()`.